packages/tupaia-web/.eslintrc.cjs

```diff
@@ -1,4 +1,4 @@
 /* eslint-env node */
 module.exports = {
-  extends: '../../.eslintrc-js-frontend.json',
+  extends: ['../../.eslintrc-js-frontend.json', 'react-app'],
```

Comment:
This is to handle the linting errors we were getting

Reply:
Great find! thanks